devtools-ci-index.yaml

```diff
@@ -2013,35 +2013,34 @@
         - vault-secrets:
             <<: *vault_defaults
             secrets:
-                - secret-path: 'devtools-osio-ci/e2e-tests-creds'
+                - secret-path: 'devtools-osio-ci/e2e/{osio_creds}'
                   secret-values:
-                    - env-var: 'OSIO_USERNAME'
-                      vault-key: '{osio_creds}_username'
                     - env-var: 'OSIO_PASSWORD'
-                      vault-key: '{osio_creds}_password'
+                      vault-key: 'password'
     beforeScript: |
         # if the PR to saas-openshiftio is done immediatelly after the commit
         # to dependency's master, it might be still deploying to prod-preview
         echo "Sleeping for 1 min"
         sleep 60
+        export OSIO_USERNAME="{osio_creds}"
```

Comment:
this doesn't work, the contents of OSIO_USERNAME will be literally `{osio_creds}`.

Reply:
What is the reason? The fact that `beforeScript` is included in `runScript`? I.e. if I include it directly to the main script it would work?